backend/inspirehep/records/api/literature.py

```diff
@@ -362,7 +362,7 @@ def import_article(identifier):
 
     elif is_doi(identifier):
         pid_type = "doi"
-        pid_value = normalize_doi(identifier)
+        pid_value = normalize_doi(identifier).lower()
```

Comment:
is it only for doi though? shouldn't it work for arxiv as well? If yes, I would just do ```identifier.lower``` before this